paper-typeahead.js

```diff
@@ -441,9 +441,8 @@ Polymer({
     if (targetResult === undefined) {
       this.fire('customvalentered', {target: this.typedValue});
     } else {
-      this.value = this._getDataItemValue(
-        targetResult, this.dataKey);
-      this.fire('selected', {target: this.value, targetResult: targetResult});
```

Comment:
Please double check nobody is relying on `targetResult` from the event payload.

Reply:
Not that I could find. This was pretty much based on the new implementation: https://github.com/nuxeo/paper-typeahead/blob/91e1829523d5336104b7db593f5650dbde272baa/paper-typeahead.js#L239